sklearn/cross_validation.py

```diff
@@ -632,11 +638,13 @@ class ShuffleSplit(BaseShuffleSplit):
     n_iter : int (default 10)
         Number of re-shuffling & splitting iterations.
 
-    test_size : float (default 0.1), int, or None
+    test_size : float (default 0.1), int, or None,
```

Comment:
Actually, the default value here is still 0.1, we've talked about that. though we set the BaseShuffleSplit __init__(test_size=None, train_size=None), the final value of test_size and train_size will be computed in _validate_shuffle_split(), where if they are both None, the value is (.1/.9). This behavior is equal to set (.1/.9) as default.